tests/Integration/tests/configuration/input/guides.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<guides xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+        links-are-relative="true"
+        default-code-language="php"
+>
+    <project title="My Project" version="3.1.4" />
+    <inventories>
+        <inventory id="h2document" url="https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/" />
+        <inventory id="t3coreapi" url="https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/" />
```

Comment:
According to https://github.com/phpDocumentor/guides/blob/main/packages/guides-cli/resources/schema/guides.xsd the `<inventory>` tag would need to have a `path` attribute, not `url`? And not `id` but `name`?

Reply:
Mmh, we are doing it like this in the tests, and it only works like that. https://github.com/phpDocumentor/guides/blob/main/tests/Integration/tests/navigation/intersphinx-link/input/guides.xml

I think we need to update the scheme in the guides and add some automatic testing against the scheme in our testing workflow in the guides...

Reply:
Oh, I didn't know they diverged. Looks like a breaking change, too, so a new xsd revision would need to be created? Do you know how phpDocumentor would handle this?

Reply:
We did not yet publish v1.0 of the guides so we can still make breaking changes without a problem.

Reply:
(Stalled for now, because we can't get the XSD validation to work properly on the existing XML+XSD pairs of the guides repo)

Reply:
I believe, this is fixed by https://github.com/phpDocumentor/guides/pull/668